src/store/configureStore.js

```diff
@@ -0,0 +1,16 @@
+import { configureStore } from '@reduxjs/toolkit'
```

Comment:
By the way, you don't have in your `package.json`. You should [install it](https://redux.js.org/introduction/getting-started#redux-toolkit) if you want your store to work 😛